frontend/rolecall/src/app/services/super-ballet-display.service.ts

```diff
@@ -0,0 +1,27 @@
+import {Injectable} from '@angular/core';
+
+
+@Injectable({providedIn: 'root'})
+export class SuperBalletDisplayService {
+  superBalletIsOpenMap = new Map<string, boolean>();
+
+  changeOpenState(uuid: string, isOpen: boolean) {
```

Comment:
What do you think about calling it "setOpenState"? It's a tiny bit shorter and a little closer aligned with what it's doing.